### PR TITLE
Fixing two new formats when people include the white/black information.

### PR DESCRIPTION
**Pull request by lakinwecker**
Fixes this: @player1 (white) v @player2 (black) mm/dd @ HH:MM

Also remove the steveharwell hack as it's no longer necessary.

### CODE REVIEW

File: test/test_spreadsheets.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ describe('scheduling', function() {
                 assert.equal(results.date.format(), expected.date);
                 assert.equal(results.white, expected.white);
                 assert.equal(results.black, expected.black);
+                assert.equal(results.black, expected.black);
```
Comment (endrawes0) [inline test/test_spreadsheets.js]: What's happening here?

Comment (endrawes0) [inline test/test_spreadsheets.js]: Bad copy paste or something?

Comment (lakinwecker) [inline test/test_spreadsheets.js]: Yup, bad copy/paste. Fixing!

**Comment by lakinwecker** [comment]
I removed that errant copy/paste

**Comment by endrawes0** [comment]
thanks - sorry for the delay - busy week at work.

**Comment by lakinwecker** [comment]
No worries.

**Pull request merged**